src/constraint/spec.ts

```diff
@@ -429,6 +444,21 @@ export const SPEC_CONSTRAINTS: SpecConstraintModel[] = [
       return true;
     }
   },
+  {
+    name: 'rawOnly',
+    description: 'Omit aggregate plots.',
+    properties: [Property.AGGREGATE, Property.AUTOCOUNT],
+    allowEnumSpecForProperties: true,
+    strict: false,
+    satisfy: (specM: SpecQueryModel, schema: Schema, opt: QueryConfig) => {
+      if (opt.rawOnly) {
```

Comment:
You don't need to check for this.  Our generation would trigger this constraints only if it is true.